src/main/java/uk/gov/ons/ctp/response/casesvc/domain/repository/CaseRepository.java

```diff
@@ -94,4 +67,12 @@
    * @return
    */
   Case findBySampleUnitId(@Param("sampleUnitId") UUID sampleUnitId);
+
+  /**
+   * Find a case by it's PK
```

Comment:
It's worth deleting the @param and @return if there is no more context to add

Reply:
fixed